tests/test_remote.py

```diff
@@ -477,6 +478,46 @@ def test_tunnel(self):
                 print(p.communicate())
                 assert data == b"hello world"
 
+
+    def test_reverse_tunnel(self):
+
+        def serve_reverse_tunnel(queue):
```

Comment:
FYI, this is not picklable, so can't be forked. Why did this only matter on Python 3.10 + macOS? I have no idea. Anyway, moving it out of the test function should fix the problem.

Reply:
That's very strange. Thanks for cleaning that up!

Reply:
Yeah I can definitely look into it!